src/components/InlineCodeBlock/WrappedText.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {Fragment} from 'react';
+import {StyleProp, TextStyle, View, ViewStyle} from 'react-native';
+import Text from '@components/Text';
+import useThemeStyles from '@styles/useThemeStyles';
+import CONST from '@src/CONST';
+import type ChildrenProps from '@src/types/utils/ChildrenProps';
+
+type WrappedTextProps = ChildrenProps & {
+    /** Style to be applied to Text */
+    textStyles?: StyleProp<TextStyle>;
+
+    /** Style for each word(Token) in the text, remember that token also includes whitespaces among words */
+    wordStyles?: StyleProp<ViewStyle>;
+};
+
+/**
+ * Breaks the text into matrix
+ * for eg: My Name  is Rajat
+ *  [
+ *    [My,' ',Name,' ',' ',is,' ',Rajat],
+ *  ]
+ */
+function getTextMatrix(text: string): string[][] {
+    return text.split('\n').map((row) => row.split(CONST.REGEX.SPACE_OR_EMOJI).filter((value) => value !== ''));
+}
+
+function WrappedText({children, wordStyles, textStyles}: WrappedTextProps) {
+    const styles = useThemeStyles();
+
+    if (typeof children !== 'string') {
+        return null;
+    }
+
+    const textMatrix = getTextMatrix(children);
+
+    return textMatrix.map((rowText, rowIndex) => (
```

Comment:
I see you removed the outer fragment `<></>` and are returning the map directly, please let's test this to ensure there is no regressions.

Reply:
Retested all of the platforms 👌